Source/SwiftLintFramework/Rules/ColonRule.swift

```diff
@@ -103,16 +103,22 @@ public struct ColonRule: CorrectableRule, ConfigurationProviderRule {
     public func correctFile(file: File) -> [Correction] {
         let matches = violationRangesInFile(file, withPattern: pattern)
         guard !matches.isEmpty else { return [] }
+        let fileregions = file.regions()
 
         let regularExpression = regex(pattern)
         let description = self.dynamicType.description
         var corrections = [Correction]()
         var contents = file.contents
         for range in matches.reverse() {
+            let location = Location(file: file, characterOffset: range.location)
+            let region = fileregions.filter {$0.contains(location)}.first
+            if region?.isRuleDisabled(self) == true {
```

Comment:
`region?.isRuleDisabled(self)` returns false when `region` is nil

```
  if region?.isRuleDisabled(self) == true {  // returns false when region is nil
                continue
            } else {...}
```

plus I was trying to stay away from double negatives ;)
https://twitter.com/NachoSoto/status/762358223122927616